sdc/functions/numpy_like.py

```diff
@@ -710,6 +710,45 @@ def dropna_impl(arr, idx, name):
     return dropna_impl
 
 
+def find_idx(arr, idx):
+    pass
+
+
+@sdc_overload(find_idx)
+def find_idx_overload(arr, idx):
+    dtype = arr.dtype
+
+    def find_idx_impl(arr, idx):
+        chunks = parallel_chunks(len(arr))
+        arr_len = numpy.empty(len(chunks), dtype=numpy.int64)
+        length = 0
+
+        for i in prange(len(chunks)):
+            chunk = chunks[i]
+            res = 0
+            for j in range(chunk.start, chunk.stop):
+                if arr[j] == idx:
+                    res += 1
+            length += res
+            arr_len[i] = res
```

Comment:
You could allocate list of list. The length of the first list is equal to number of chunks. In this case you could safely use `append` for list related to chunk. So in this case single loop would be enough. (and another loop (probably, not parallel) to merge all lists into one)